README.rst

```diff
@@ -41,13 +41,32 @@
 MAPIE - Model Agnostic Prediction Interval Estimator
 ====================================================
 
-**MAPIE** allows you to easily estimate prediction intervals (or prediction sets)
-using your favourite scikit-learn-compatible model for
-single-output regression or multi-class classification settings.
+Quantifying the uncertainties and controlling the risks of ML model predictions is of crucial importance
+for developing and deploying reliable artificial intelligence (AI) systems. Uncertainty quantification (UQ)
+involves all the stakeholders who develop and use AI models.
 
-Prediction intervals output by **MAPIE** encompass both aleatoric and epistemic
-uncertainties and are backed by strong theoretical guarantees thanks to conformal
-prediction methods [1-7].
+**MAPIE** is an open-source Python library hosted on scikit-learn-contrib project that allows you to:
+
+- easily **estimate conformal prediction intervals** (or prediction sets) given a degree of confidence or risk
+  for single-output regression, binary and multi-class classification settings [3-9].
+- easily **control risks** (such as coverage, recall or any other non-monotone risk) by estimating
+  relevant prediction sets for multi-label classification and beyond [10-12].
+- easily **wrap your favorite scikit-learn-compatible model** for the purposes just mentioned.
+
+**MAPIE** has been designed to respect three fundamental pillars:
+
+- Implemented methods are **model and use case agnostic** in order to address all relevant use cases tackled in industry.
+- Implemented methods must have **strong theoretical guarantees** on the marginal coverage of the estimated uncertainties
+  with as little assumption on the data or the model as possible.
+- Implemented methods follow **state-of-the-art trends** that respect programming standards in order to develop trustworthy AI systems.
+
+Importantly, **MAPIE** contributes to the wide diffusion of the attractive **Conformal Prediction** (CP) framework for regression
+and classification settings that is model and use case agnostic with mathematical guarantees on the marginal coverages on the prediction sets
+with few assumptions (distribution-free and data-exchangeability assumptions) [1-2].
+
+Prediction sets output by **MAPIE** encompass both aleatoric and epistemic uncertainties
+and are backed by strong theoretical guarantees using a variety of conformal prediction methods [3-9]
+and a variaty of conformal risk control methods [10-12].
 
 
 🔗 Requirements
```

Comment:
Why don't we add something about the predictions set for classification?

Reply:
I will make a proposition.